atlas/commands/bootstrap/templates/Makefile.gotmpl

```diff
@@ -74,9 +77,9 @@ clean:
 {{ if .WithDatabase }}
 .PHONY: migrate-up
 migrate-up:
-	@migrate -database 'postgres://$(DATABASE_ADDRESS)/{{ .Name | Database }}?sslmode=disable' -path ./db/migrations up
+	@migrate -database 'postgres://$(DATABASE_USERNAME):$(DATABASE_PASSWORD)@$(DATABASE_ADDRESS)/{{ .Name | Database }}?sslmode=disable' -path ./db/migrations up
```

Comment:
I would prefer having something like:
```
DATABASE_NAME={{ .Name | Database }}
DATABASE_URL?=postgres://$(DATABASE_USERNAME):$(DATABASE_PASSWORD)@$(DATABASE_ADDRESS)/$(DATABASE_NAME)?sslmode=disable
```
at the top and use `DATABASE_URL` in migrate up/down.
Also I think it makes more sense to do migration running `migrate` image in docker, similar to what we do for gentool. This way we do not force installing migrate.